runtime/server/catalog.go

```diff
@@ -5,7 +5,9 @@ import (
 	"fmt"
 	"strings"
 
+	"github.com/mohae/deepcopy"
```

Comment:
This does not seem like a reliable dependency

Reply:
You can leverage `proto.Clone(m)` on the actual `runtimev1.MetricsView` object instead